influxdb3/tests/server/system_tables.rs

```diff
@@ -185,12 +185,12 @@ async fn last_caches_table() {
         let batches = collect_stream(resp).await;
         assert_batches_sorted_eq!(
             [
-                "+-------+---------------------+----------------+---------------+-------+-------+",
-                "| table | name                | key_columns    | value_columns | count | ttl   |",
-                "+-------+---------------------+----------------+---------------+-------+-------+",
-                "| cpu   | cpu_host_last_cache | [host]         |               | 1     | 14400 |",
-                "| mem   | mem_last_cache      | [host, region] | [time, usage] | 1     | 60    |",
-                "+-------+---------------------+----------------+---------------+-------+-------+",
+                "+-------+---------------------+-------------+---------------+-------+-------+",
+                "| table | name                | key_columns | value_columns | count | ttl   |",
+                "+-------+---------------------+-------------+---------------+-------+-------+",
+                "| cpu   | cpu_host_last_cache | [1]         |               | 1     | 14400 |",
```

Comment:
This can come in a follow on PR, but since the system tables are for humans to read, I think it would be much better to have the id and name for the key column and value columns here.

Reply:
I will open an issue for this - agreed.

Reply:
Opened https://github.com/influxdata/influxdb/issues/25511